charts/wga/values.yaml

```diff
@@ -7,10 +7,23 @@
 ## @param global.imageRegistry Global Docker image registry
 global:
   imageRegistry: "ghcr.io/kraudcloud"
+  privateImageRegistry: "ctr.0x.pt"
 
 ## @section Common parameters
 ##
 
+welcome:
+  image: 
+    name: "wga-welcome"
+    tag: "main"
+    pullPolicy: ""
+
+  service:
+    type: "ClusterIP"
+
+  annotations: {}
+  labels: {}
+
```

Comment:
nit: I would put this in the unbound block (bottom of the file in this case), so it looks like this. Makes it clearer it's part of the dns section of the app

```
unbound:
  welcomeImage: 
    name: "wga-welcome"
    tag: "1.0.0"
    pullPolicy: ""
```